### PR TITLE
ReleaseNotes: Updated changelog and release notes for 7.3.5

### PR DESCRIPTION
**Pull request by grafanabot**
This exciting! So much has changed!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -54,10 +54,17 @@ In order not to break all angular panel plugins and data sources we have some cu
...
+### Features and enhancements
+
+* **Alerting**: Improve Prometheus Alert Rule error message. [#29390](https://github.com/grafana/grafana/pull/29390), [@wbrowne](https://github.com/wbrowne)
+
 ### Bug fixes
 
+* **Alerting**: Fix alarm message formatting in Dingding. [#29482](https://github.com/grafana/grafana/pull/29482), [@tomowang](https://github.com/tomowang)
 * **AzureMonitor**: Fix unit translation for MilliSeconds. [#29399](https://github.com/grafana/grafana/pull/29399), [@secustor](https://github.com/secustor)
 * **Instrumentation**: Fix bug with invalid handler label value for HTTP request metrics. [#29529](https://github.com/grafana/grafana/pull/29529), [@bergquist](https://github.com/bergquist)
+* **Prometheus**: Fix display name field has no effect. [#29441](https://github.com/grafana/grafana/pull/29441), [@zoltanbedi](https://github.com/zoltanbedi)
```
Comment (achatterjee-grafana) [inline CHANGELOG.md:66]: Not sure what "Fix display name field has no effect." means. Can this be a little more clear? Thank you.

**Review by achatterjee-grafana — APPROVED** [review]
Looks good, one request for greater clarity.

File: CHANGELOG.md (hunk 1 above)
Comment (wbrowne) [reply CHANGELOG.md:66]: @zoltanbedi

Comment (zoltanbedi) [reply CHANGELOG.md:66]: I really have no idea what else could be in the there to be more clear. Any suggestions?

Comment (torkelo) [reply CHANGELOG.md:66]: updated PR, with slightly more descriptive title: "Prometheus: Fixes problem where changing display name in Field tab had no effect"

Comment (matthewhelmke) [reply CHANGELOG.md:66]:
```suggestion
* **Prometheus**: Fix bug where display name field has no effect. [#29441](https://github.com/grafana/grafana/pull/29441), [@zoltanbedi](https://github.com/zoltanbedi)
```

Comment (matthewhelmke) [reply CHANGELOG.md:66]: Or this
```suggestion
* **Prometheus**: Fix issue where display name field has no effect. [#29441](https://github.com/grafana/grafana/pull/29441), [@zoltanbedi](https://github.com/zoltanbedi)
```

Comment (achatterjee-grafana) [reply CHANGELOG.md:66]: How about "Fix issue where the changing the field display name was not having any effect". Hope I understood the issue (https://github.com/grafana/grafana/issues/29437) correctly and this helps.

**Comment by oddlittlebird** [comment]
@wbrowne , why was this closed rather than merged?

**Comment by wbrowne** [comment]
@oddlittlebird Due to changelog generation being automated on Github, if you need to make a change, you have to delete the existing PR and re-generate. So the changes have indeed been merged, but in a [follow-up PR](https://github.com/grafana/grafana/pull/29753)

**Pull request closed without merging**